README.md

```diff
@@ -31,6 +31,17 @@ If you are the current maintainer of the package then create a new request using
 * The issue will get labelled as assigned, awaiting approval
 * When the package gets approved by the moderation process, close the issue (or ask for it to be closed).
 
+When you pick up a request for either a new package or a new maintainer, remember to keep the maintainers of this repository updated on the current status of the work being done.
+A comment from the one picking up a request is expected to be done at the following stages of a request:
+
+1. When you start working on the package, or get added as a new maintainer (issue will then be labeled as `Status: In Progress` and you will be assigned to the issue).
+2. When the new package or updated version is pushed to chocolatey (issue will then be moved to `Status: Review`).
```

Comment:
Chocolatey instead of chocolatey?

Reply:
it was actually meant to be `chocolatey.org` (forgot that last part).
Not sure what would be best, `Chocolatey` or `chocolatey.org`

Reply:
I think chocolatey.org

Reply:
Changed this to `chocolatey.org`